src/main/java/pl/edu/agh/firecell/model/Cell.java

```diff
@@ -7,6 +7,14 @@ public record Cell(
         Material material
         ) {
 
+        public boolean isSolid() {
+                return material.getMatterState().equals(MatterState.SOLID);
```

Comment:
I'm not sure what happened with this file but I see 8-space indents :D Other files have 4-space indents